mde-services/src/main/java/de/terrestris/mde/mde_backend/properties/KeycloakProperties.java

```diff
@@ -0,0 +1,40 @@
+package de.terrestris.mde.mde_backend.properties;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+@Data
+@Configuration
+@ConfigurationProperties(prefix = "keycloak")
+@Component
+public class KeycloakProperties {
+
+  private Boolean enabled;
+
+  private String serverUrl;
+
+  private String clientSecret;
+
+  private String username;
+
+  private String password;
+
+  private String masterRealm;
+
+  private String adminClientId;
+
+  private String realm;
+
+  private String clientId;
+
+  private String principalAttribute;
+
+  private Boolean disableHostnameVerification;
+
+  private Boolean extractRolesFromResource = true;
```

Comment:
This property can potentially be removed.

Reply:
[42b4b43](https://github.com/gdi-be/mde-backend/pull/42/commits/42b4b4366270ba6cd3f9168c2e0f7eade8e534f5)